com.unity.formats.fbx/Editor/ExportSettings.cs.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 03d96513d037ea843ab436321bab4c33
+guid: f87a2d71b287d4b579dbf0858cda24d9
```

Comment:
would it be possible to keep the same guid? we used to have issues with changing them. Not sure if it would still be a problem, but just to be safe.